src/main/java/kr/pickple/back/crew/controller/CrewController.java

```diff
@@ -38,4 +43,13 @@ public ResponseEntity<CrewProfileResponse> findCrewById(
         return ResponseEntity.status(OK)
                 .body(crewService.findCrewById(crewId));
     }
+
+    @PostMapping("/{crewId}/members")
+    public void applyForCrewMemberShip(
+            @PathVariable("crewId") final Long crewId,
+            @Valid @RequestBody final CrewApplyRequest crewApplyRequest
+    ) {
+        crewMemberService.applyForCrewMemberShip(crewId, crewApplyRequest);
+        ResponseEntity.status(NO_CONTENT).build();
+    }
```

Comment:
`p1`
현재 작성하신 코드로 반환하게되면 상태코드가 반영되지 않습니다!
아래와 같이 반환 값을 ResponseEntity<Void>로 해주시고 return 해주시면 될 것 같습니다!

```suggestion
    public ResponseEntity<Void> applyForCrewMemberShip(
            @PathVariable("crewId") final Long crewId,
            @Valid @RequestBody final CrewApplyRequest crewApplyRequest
    ) {
        crewMemberService.applyForCrewMemberShip(crewId, crewApplyRequest);
        return ResponseEntity.status(NO_CONTENT).build();
    }
```